RadarSDK/Radar.m

```diff
@@ -340,9 +340,41 @@ + (void)rejectEventId:(NSString *)eventId {
     [[RadarAPIClient sharedInstance] verifyEventId:eventId verification:RadarEventVerificationReject verifiedPlaceId:nil];
 }
 
++ (void)sendLogConversionRequestWithName:(NSString * _Nonnull) name
+                                metadata:(NSDictionary * _Nullable) metadata
+                                location:(CLLocation * _Nullable) location
+                                  events:(NSArray<RadarEvent *> * _Nullable) events
+                                    user:(RadarUser * _Nullable) user
```

Comment:
based on decision yesterday, let's remove the option to send a hardcoded location